coverage/sorter.js

```diff
@@ -0,0 +1,158 @@
+var addSorting = (function () {
```

Comment:
`coverage folder` seems like unused, right?

Reply:
I'll remove the js stuff, was added by accident